index.bs

```diff
@@ -1429,9 +1429,6 @@ to have bindings in other programming languages.
 
 <h3 id="js-rtc">Preserve run-to-completion semantics</h3>
 
-Don't modify data accessed via JavaScript APIs
-while a JavaScript <a>event loop</a> is running.
-
```

Comment:
I think that we need a positive statement for this principle.

Maybe

> Propagate changes to state
> that originate outside of JavaScript execution context
> between tasks, not within them.

Reply:
Yeah, or maybe by moving up the "a task should be queued" suggestion below...

> If a change to state originates outside of the JavaScript execution context,
> propagate that change to JavaScript between tasks,
> for example by [[html#queuing-tasks|queuing a task]],
> or as part of [=update the rendering=].